conda-recipes/D47188-svml.patch

```diff
@@ -1,36 +1,5 @@
-From cd3867b657afce84bc6af48f20f2d693afc72e28 Mon Sep 17 00:00:00 2001
-From: Stuart Archibald <sarchibald@anaconda.com>
-Date: Fri, 23 Mar 2018 11:21:58 +0000
-Subject: [PATCH] Enable Intel SVML transcendental math functions.
-
-This patch enables Intel SVML-enabled auto-vectorization for all
-the transcendental math functions.
----
- include/llvm/Analysis/TargetLibraryInfo.h       |  17 +++-
- include/llvm/IR/CMakeLists.txt                  |   4 +
- include/llvm/IR/CallingConv.h                   |   3 +
- include/llvm/IR/SVML.td                         |  58 ++++++++++++
- lib/Analysis/CMakeLists.txt                     |   1 +
- lib/Analysis/TargetLibraryInfo.cpp              | 106 ++++------------------
- lib/AsmParser/LLLexer.cpp                       |   1 +
- lib/AsmParser/LLParser.cpp                      |   2 +
- lib/AsmParser/LLToken.h                         |   1 +
- lib/IR/AsmWriter.cpp                            |   1 +
- lib/IR/Verifier.cpp                             |   1 +
- lib/Target/X86/X86CallingConv.td                |  59 +++++++++++-
- lib/Target/X86/X86ISelLowering.cpp              |   3 +-
- lib/Target/X86/X86RegisterInfo.cpp              |  34 +++++++
- lib/Target/X86/X86Subtarget.h                   |   1 +
- lib/Transforms/Vectorize/LoopVectorize.cpp      |   8 +-
- test/Transforms/LoopVectorize/X86/svml-calls.ll |  40 +++++++++
- utils/TableGen/CMakeLists.txt                   |   1 +
- utils/TableGen/SVMLEmitter.cpp                  | 114 ++++++++++++++++++++++++
- utils/TableGen/TableGen.cpp                     |   8 +-
- utils/TableGen/TableGenBackends.h               |   1 +
- utils/vim/syntax/llvm.vim                       |   1 +
- 22 files changed, 366 insertions(+), 99 deletions(-)
- create mode 100644 include/llvm/IR/SVML.td
- create mode 100644 utils/TableGen/SVMLEmitter.cpp
+From https://reviews.llvm.org/D47188 rebased on top of LLVM 6.0.0
```

Comment:
It would be nice if this remained as a git formatted patch.

Reply:
I'm not sure how to add these comments in git-formatted patch. Or do you suggest to drop them away? it does not look like it breaks anything, many other patches have arbitrary headers